openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -1031,31 +1065,45 @@ message BidRequest {
     // URL of the page where the impression will be shown.
     optional string page = 7;
 
-    // Indicates if the site has a privacy policy, where 0 = no, 1 = yes.
-    optional bool privacypolicy = 8;
-
     // Referrer URL that caused navigation to the current page.
     optional string ref = 9;
 
     // Search string that caused navigation to the current page.
     optional string search = 10;
 
+    // Indicates if the site has been programmed to optimize layout
+    // when viewed on mobile devices, where 0 = no, 1 = yes.
+    optional bool mobile = 15;
```

Comment:
Do you know why 14 is not present, was it used for something since deleted? (If yes, I want to `reserve` it.)